README.md

```diff
@@ -72,12 +72,14 @@ This will create an OpenTelemetry Collector instance named `simplest`, exposing
 
 The `config` node holds the `YAML` that should be passed down as-is to the underlying OpenTelemetry Collector instances. Refer to the [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector) documentation for a reference of the possible entries.
 
-> 🚨 **NOTE:** At this point, the Operator does _not_ validate the contents of the configuration file: if the configuration is invalid, the instance will still be created but the underlying OpenTelemetry Collector might crash.
-
 > 🚨 **Note:** For private GKE clusters, you will need to either add a firewall rule that allows master nodes access to port `9443/tcp` on worker nodes, or change the existing rule that allows access to port `80/tcp`, `443/tcp` and `10254/tcp` to also allow access to port `9443/tcp`. More information can be found in the [Official GCP Documentation](https://cloud.google.com/load-balancing/docs/tcp/setting-up-tcp#config-hc-firewall). See the [GKE documentation](https://cloud.google.com/kubernetes-engine/docs/how-to/private-clusters#add_firewall_rules) on adding rules and the [Kubernetes issue](https://github.com/kubernetes/kubernetes/issues/79739) for more detail.
 
-The Operator does examine the configuration file to discover configured receivers and their ports. If it finds receivers with ports, it creates a pair of kubernetes services, one headless, exposing those ports within the cluster. The headless service contains a `service.beta.openshift.io/serving-cert-secret-name` annotation that will cause OpenShift to create a secret containing a certificate and key. This secret can be mounted as a volume and the certificate and key used in those receivers' TLS configurations.
+The Operator does examine the configuration file for a few purposes:
+
+- To discover configured receivers, services and their ports. If it finds receivers with ports, it creates a pair of kubernetes services, one headless, exposing those ports within the cluster. If the port is using environment variable expansion or cannot be parsed, an error will be returned. To ignore service creation set `observability.metrics.enableMetric` to `false`. You can create the Service objects yourself if you need them. The headless service contains a `service.beta.openshift.io/serving-cert-secret-name` annotation that will cause OpenShift to create a secret containing a certificate and key. This secret can be mounted as a volume and the certificate and key used in those receivers' TLS configurations.
```

Comment:
I believe you mixed the idea of enabling the Monitoring Service and how the Operator validates the configuration file's contents. Please try to clarify each of the topics mentioned in the doc.

Reply:
Why mixed, could you explain a bit better please? There are two separate points there:

The first list item, which talks about the receivers and theirs ports stays almost the same. The only thing I modified there is that `configured receivers and their ports` became `configured receivers, services and their ports`, which I forgot to fix. I will revert back to the original text.

The second list item is where the explanation regarding the metrics service address is explained.

Reply:
With the changes in [62e7f1e](https://github.com/open-telemetry/opentelemetry-operator/pull/3531/commits/62e7f1e2151e095f855831e514ec69314fce3f61), the 1st bullet point there has the exact same text as before.

Reply:
My point is because this setence `.. To ignore service creation set observability.metrics.enableMetric to false.` is placed in the same bullet point of the description how the operator handles the receivers and their ports.

Reply:
Ahhh, my bad. Another bad leftover there. I'll get rid of it.

Reply:
Should be good now after [a4ef4f0](https://github.com/open-telemetry/opentelemetry-operator/pull/3531/commits/a4ef4f05a4b967f605ea056c3ebcd878c6c3610a)